boards/nxp/mimxrt1180_evk/doc/index.rst

```diff
@@ -109,6 +109,8 @@ configuration supports the following hardware features:
 +-----------+------------+-------------------------------------+
 | ACMP      | on-chip    | analog comparator                   |
```

Comment:
I also noticed that this is wrong, the driver/component of acmp is `sensor`

Reply:
Update ACMP description for all wrong NXP platforms.